workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/WorkerInspectorWindow.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.Editor.UIElements;
+using Unity.Entities;
+using UnityEditor;
+using UnityEngine.UIElements;
+
+namespace Improbable.Gdk.Debug.WorkerInspector
+{
+    internal class WorkerInspectorWindow : EditorWindow
+    {
+        private WorldSelector worldSelector;
+        private EntityList entityList;
+
+        [MenuItem("SpatialOS/Window/Worker Inspector", false)]
+        public static void ShowWindow()
+        {
+            var windowType = typeof(EditorWindow).Assembly.GetType("UnityEditor.InspetorWindow");
+            var window = GetWindow<WorkerInspectorWindow>(windowType);
+            window.titleContent.text = "Worker Inspector";
+            window.Show();
+        }
+
+        private void OnEnable()
+        {
+            SetupUI();
+            worldSelector.UpdateWorldSelection();
+        }
+
+        private void OnInspectorUpdate()
+        {
+            worldSelector.UpdateWorldSelection();
+            entityList.Update();
+        }
+
+        private void SetupUI()
+        {
+            const string windowUxmlPath = "Packages/io.improbable.gdk.debug/WorkerInspector/Templates/WorkerInspectorWindow.uxml";
+            const string windowUssPath = "Packages/io.improbable.gdk.debug/WorkerInspector/Templates/WorkerInspectorWindow.uss";
+
+            var windowTemplate = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(windowUxmlPath);
+            windowTemplate.CloneTree(rootVisualElement);
+
+            var stylesheet = AssetDatabase.LoadAssetAtPath<StyleSheet>(windowUssPath);
+            rootVisualElement.styleSheets.Add(stylesheet);
+
+            worldSelector = rootVisualElement.Q<WorldSelector>();
+            worldSelector.OnWorldChanged += OnWorldChanged;
```

Comment:
Do we need to manually remove these callbacks if the window is closed?

Reply:
If the window is closed, the world selector element is removed along with it, so its all GC'ed